kubernetes/config/kube_config.py

```diff
@@ -22,151 +22,243 @@
 from kubernetes.client import configuration
 from oauth2client.client import GoogleCredentials
 
-_temp_files = []
+from .incluster_config import ConfigException
+
+_temp_files = {}
 
 
 def _cleanup_temp_files():
-    for f in _temp_files:
-        os.remove(f)
+    for temp_file in _temp_files.values():
+        os.remove(temp_file)
```

Comment:
Spurious change, or just OCD? :smile:

This is functionally 100% sans a better for loop variable name. The `values()` isn't necessary.